src/Samples/NetCoreClient/Program.cs

```diff
@@ -6,16 +6,17 @@ namespace NetCoreClient
 {
     class Program
     {
-        private static readonly string s_hostname = "localhost";
-
-        static void Main()
+        /// <param name="hostname">use commanline argument --hostname localhost
+        /// or something similar to indicate the WCF Server hostname </param>
+        static void Main(string hostname)
```

Comment:
I'm not comfortable pulling in an alpha release package for command line parsing for a single parameter. I would rather just check if args.Length == 1, then set hostname to that. So something like:
```c#
string hostname = args.Length == 1 ? args[0] : null;
```
It's a nice concept, but I had a look at their code and there's a few problems, like consuming an extra thread unnecessarily for the duration of the app. I know this is samples, but I don't want to encourage other people to use an alpha package.

Reply:
Makes sense, I'll remove it